key_event.js

```diff
@@ -0,0 +1,21 @@
+if (window == top) {
+    window.addEventListener('keyup', doKeyPress, false); //add the keyboard handler
+    }
+    
+    var acc_num = [0,1,2,3,4,5,6,7,8,9]
+    function doKeyPress(e){
+	var n = parseInt(e.keyCode)-48
+        if (((e.shiftKey&&e.altKey)||e.altKey)&&acc_num.includes(n)){
```

Comment:
Here as well

Reply:
I have made the required changes, waiting for further comments.